cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -119,6 +120,9 @@ General:
     WARNING: this option disables ALL logging output (including connection
     errors), which will likely make debugging difficult. The -quiet flag takes
     precedence over the -verbose flag.
+  -log_debug_stdout
+    When explicitly set to true, verbose and info log messages will be directed
+	stdout as a pose to the default stderr.
```

Comment:
'to stdout as opposed to the default stderr'

Reply:
Just pushed a fix